src/goto/cli.py

```diff
@@ -185,12 +210,14 @@ def print_help():
     'rmprofile': 'Remove a profile',
     'profile': 'Switch to a (possibly non-existant) profile',
     'profiles': 'List all profiles',
-    'install': 'Install goto for the given shell, "bash" or "zsh"'
+    'install': 'Install goto for the given shell, "bash" or "zsh"',
+    'config': 'Configure behaviour get/set/remove'
 }
 
 BASH_ZSH = click.Choice(['bash', 'zsh'])
 
 @click.command()
+@click.argument('configuration', nargs=-1)
```

Comment:
nargs is a parameter for options but cannot be < 0, which we need in this case for variable length. This can only be done through vardiadic arguments, at least to my understanding.

Reply:
Sorry about the many stupid issues! I'm still learning :P

Reply:
I made a quick click program and verified this, yeah. click doesn't allow -1 for `nargs`

```
import click

@click.command()
@click.option('--config', '-c', nargs=-1)
def main(**kwargs):
    print('Hello!', kwargs)

if __name__ == "__main__":
    main()
```

I think it would be nicer to have like --config-set, --config-get and --config-remove and we can specify exactly how many things and what types should be added but I think that can be done later down the line. For now this can use variadiac arguments, it's good enough!

I'll merge this into master! Congrats on being the first outside contributor to goto!

Reply:
Thank you for your insights, it's been a pleasure. Excited to work on more for goto!